packages/deploy-script-support/src/coreProposalBehavior.js

```diff
@@ -22,7 +22,7 @@ export const permits = {
  * definitions.
  *
  * @param {object} opts
- * @param {string} opts.manifestInstallRef
+ * @param {{ bundleName?: string, bundleID?: string}} opts.manifestInstallRef
```

Comment:
are there really 4 possibilities? Or did you mean:

```suggestion
 * @param {{ bundleName: string } | { bundleID: string }} opts.manifestInstallRef
```

Reply:
yep, that's right.. mfig pointed me at a way to satisfy tsc (but it's kinda ugly), see what you think